authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,38 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
 
-"""Configure JWT Here"""
+from django.conf import settings
 
+from rest_framework import authentication, exceptions
+
+from .models import User
+
+from authors.apps.authentication.auth_token import AuthenticationToken
+
+authentication_token = AuthenticationToken()
+
+class JWTAuthentication(authentication.BaseAuthentication):
+    authentication_header_prefix = "Bearer"
+    def authenticate(self, request):
+        auth_headers = authentication.get_authorization_header(request).split()
+        if not auth_headers:
+            return None
+        if len(auth_headers) == 1:
+            return None
+        if len(auth_headers) > 2:
+            return None
```

Comment:
I don't know if it's me who's not understanding the real significance of lines from 17 to 22 but I think you could replace them with a single if-statement. Like:
```
if len(auth_headers) != 2:
            return None
```
This uses much less computing resources when the app is running compared to having three if-statements.

Reply:
I have refactored it. thanks for the recommendation